apps/mitt-konto/src/MittKonto/Routes.purs

```diff
@@ -0,0 +1,48 @@
+module MittKonto.Routes where
+
+import Prelude hiding ((/))
+
+import Data.Either (note)
+import Data.Generic.Rep (class Generic, NoArguments(..))
+import KSF.Api.Subscription (Subsno)
+import KSF.Api.Subscription (fromString, toString) as Subsno
+import Routing.Duplex (RouteDuplex(..), RouteDuplex', as, end, int, param, path, root, segment, suffix)
+import Routing.Duplex.Generic as G
+import Routing.Duplex.Generic.Syntax ((/))
+import Routing.Duplex.Parser (RouteParser(..), RouteError(..), RouteResult(..))
+import Routing.Duplex.Printer as R
+
+data MittKontoRoute
+  = InvoiceDetailPage Int
```

Comment:
I think it's actually nice, because a wild `InvoiceDetail` somewhere in the code doesn't really mean anything, whereas `InvoiceDetailPage` puts a lot of context to it. I think a plain `InvoiceDetail` would lead the occasional code reader to think about a subscription or a payment, but no way a route.

Reply:
Yeah, let's drop them. There's always the option to import `MittKonto.Routes` as qualified if they clash.